onnxruntime/core/graph/graph.cc

```diff
@@ -6,6 +6,7 @@
 #pragma warning(disable : 4244)
 #endif
 
+#include <cassert>
```

Comment:
Do you need this still?

Reply:
There is one (previously existing) usage of assert in this file.